spec/system/consumer/shopping/shopping_spec.rb

```diff
@@ -165,8 +165,10 @@
         let(:exchange2) { oc2.exchanges.to_enterprises(distributor).outgoing.first }
 
         before do
+          pending("Issue #12076")
           exchange1.update_attribute :pickup_time, "frogs"
           exchange2.update_attribute :pickup_time, "turtles"
+          distributor.update!(preferred_shopfront_message: "Hello!")
```

Comment:
We might wonder in the future why this is set. Let's add a comment when we solve this issue.